packages/@coorpacademy-components/src/organism/list-items/index.js

```diff
@@ -8,7 +8,7 @@ import style from './style.css';
 const ListItems = ({title, buttonLink, items, 'aria-label': ariaLabel}) => {
   const itemsView = items.map((item, index) => (
     <li key={item.id} className={style.item} data-name={`content-${index}`}>
-      <ListItem {...item} />
+      <ListItem {...item} order={index} aria-label={ariaLabel} />
```

Comment:
```suggestion
      <ListItem {...item} order={index} />
```
le aria-label est déjà dans `item`